Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.yml

```diff
@@ -122,6 +122,13 @@ script:
       name: sid
       required: true
       secret: false
+    - default: false
+      defaultValue: '100'
+      description: The maximum number of returned results per search. To retrieve all results, enter "0" (Not recommended).
+      isArray: false
+      name: count
```

Comment:
Wouldn't `limit` be a more appropriate name here? since it limits the number of results rather than count them

Reply:
I agree, changing